packages/contracts/contracts/Unirep.sol

```diff
@@ -46,6 +46,8 @@ contract Unirep is IUnirep, VerifySignature {
     uint8 public immutable numEpochKeyNoncePerEpoch;
     uint8 public immutable replNonceBits;
 
+    uint48 public attestationCounter;
```

Comment:
I think let's call this `attestationCount`. I think it makes more sense when accessing it.